crates/connector_configs/toml/development.toml

```diff
@@ -2240,6 +2240,8 @@ type="Text"
 api_key="Product Activation Key"
 key1="Payment Access Key"
 api_secret="Tariff ID"
+[novalnet.connector_webhook_details]
+merchant_secret="Source verification key"
```

Comment:
The connector specific key here is "Payment Access Key", since I have already mentioned that in key1, writing "Payment Access Key" again, wont it be a little confusing?  🤔